aws-cli/bash-linux/s3/metadata.yaml

```diff
@@ -6,14 +6,9 @@ files:
     apis:
       - service: s3
         operations:
-          - mb
-          - ls
-          - rm
-          - rb
-      - service: s3api
-        operations:
-          - wait bucket-exists
-          - wait bucket-not-exists
-          - wait object-exists
-          - wait object-not-exists
-          - put-object
+          - create-bucket
```

Comment:
Operations should use the same name as their underlying HTTP request/response counterpart.

Reply:
Whoops!  Yeah, fixed. :-)